### PR TITLE
Add missing license header and usubscription export

### PR DESCRIPTION
**Pull request by AnotherDaniel**
The recent introduction of the ::core submodule structure missed making the Update element visible to users...

### CODE REVIEW

File: src/core/udiscovery.rs [hunk 1]
```diff
@@ -1,3 +1,16 @@
+/********************************************************************************
+ * Copyright (c) 2023 Contributors to the Eclipse Foundation
```
Comment (sophokles73) [inline src/core/udiscovery.rs:2]: 2024?

File: src/core/usubscription.rs [hunk 2]
```diff
@@ -1,9 +1,23 @@
+/********************************************************************************
+ * Copyright (c) 2023 Contributors to the Eclipse Foundation
```
Comment (sophokles73) [inline src/core/usubscription.rs:2]: 2024?

File: src/core/utwin.rs [hunk 3]
```diff
@@ -1 +1,14 @@
+/********************************************************************************
+ * Copyright (c) 2023 Contributors to the Eclipse Foundation
```
Comment (sophokles73) [inline src/core/utwin.rs:2]: 2024?

**Review by sophokles73 — CHANGES_REQUESTED** [review]

File: src/core/utwin.rs (hunk 3 above)
Comment (AnotherDaniel) [reply src/core/utwin.rs]: fixed

File: src/core/usubscription.rs (hunk 2 above)
Comment (AnotherDaniel) [reply src/core/usubscription.rs]: fixed

File: src/core/udiscovery.rs (hunk 1 above)
Comment (AnotherDaniel) [reply src/core/udiscovery.rs]: fixed

**Review by PLeVasseur — APPROVED** [review]
Thanks for catching this!

**Review by sophokles73 — APPROVED** [review]
LGTM

**Pull request merged**